content/en/docs/tasks/observability/kiali/index.md

```diff
@@ -184,11 +184,54 @@ $ oc patch clusterrole kiali -p '[{"op":"add", "path":"/rules/-", "value":{"apiG
 
         {{< image width="70%" link="./kiali-service-graph.png" caption="Example Service Graph" >}}
 
-1. To examine the details about the Istio configuration, click on the
-   **Applications**, **Workloads**, and **Services** menu icons on the left menu
-   bar. The following screenshot shows the Bookinfo applications information:
+## Examining Istio configuration
 
-   {{< image width="80%" link="./kiali-services.png" caption="Example Details" >}}
+1.  To examine the details about the Istio configuration, click on the
+    **Applications**, **Workloads**, and **Services** menu icons on the left menu
+    bar. The following screenshot shows the Bookinfo applications information:
+
+    {{< image width="80%" link="./kiali-services.png" caption="Example Details" >}}
+
+## Creating weighted routes
+
+Create a weighted route using the Kiali weighted route wizard.
+This allows you to route a specific percentage of request traffic to your individual workloads.
+
+1.  View the `Versioned app graph` of the `bookinfo` graph.
+
+    *   Make sure you have selected `Requests percentage` in the `Edge Labels` drop down menu
+        to see the percentage of traffic routed to each workload.
+
+    *   Make sure you have selected the `Service Nodes` check box in the `Display` drop down menu
+        to view the service nodes in the graph.
+
+    {{< image width="80%" link="./kiali-wiz0-graph-options.png" caption="Bookinfo Graph Options" >}}
+
+1.  Focus on the `ratings` service within the `bookinfo` graph by clicking on the `ratings` service (triangle) node.
```

Comment:
It looks like these first two steps are just viewing the current configuration.  We should probably make it clear that these steps are not required to create a weighted route, but are just to view the current configuration.
One way to do that is to preface the step with (Optional).

Reply:
I'm not sure about changing this here. Every step in the Task aren't necessarily documenting all required steps to use a particular feature in general. These are just steps the user is required to do to complete the Task being documented (and a Task in Istio's documentation is not a fully documenting a feature - a Task is just stepping the user through examples and demos - at least, that is how I understand Istio's Task documentation).

In this case, these steps are required to complete the Task even though the user won't necessary do this when (in general) they create routes via the wizard.

In other words, this Task page isn't documenting how, in general, one uses the wizard to create routes. It's just a Task guide to walk the user through examples.

Reply:
I was thinking that if someone was consulting the docs to see how to create a route, those steps aren't necessary.  But if this is more of a tutorial than a procedure, then this is fine.

Reply:
> if this is more of a tutorial

THAT'S the word I was looking for -- "tutorial" :) Yes, this Kiali Task is more of a tutorial.